src/services/handleFilters.js

```diff
@@ -0,0 +1,45 @@
+const handleFilters = (sortType, page = '') => {
+    let temp = sortType;
+
+    if (page === 'users' && temp === 'Name') {
+        temp = 'Username';
+    } else if (page === 'users' && temp === 'Popular') {
+        temp = 'Popular users';
+    }
+
+    let handleToday = function(a) {
+        const aDate = Date.now() - new Date(a.created_at).getTime()
+        return aDate < 86300000 && aDate > 1000
+    }
+
+    let handleWeek = function(a) {
+        const aDate = Date.now() - new Date(a.created_at).getTime()
+        return aDate < 604800000
+    }
+
+    let handleMonth = function(a) {
+        const aDate = Date.now() - new Date(a.created_at).getTime()
+        return aDate < 2628000000
+    }
+
+    let handleYear = function(a) {
+        const aDate = Date.now() - new Date(a.created_at).getTime()
+        return aDate < 31540000000
+    }
```

Comment:
Can create descriptive constants for,
86300000, 604800000, 2628000000, and 31540000000

Reply:
Sorry, I didn't get your point. Do you mean make variables of them and write the variables in condition?

Reply:
@BurhanRaja yes exactly